src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1439,7 +1439,13 @@ internal enum ErrorCode
         ERR_DelegateRefMismatch = 8189,
         #endregion stragglers for C# 7
 
-        // Available  = 8190-8195
+        #region diagnostics for parse options
+        WRN_BadSourceCodeKind = 8190,
```

Comment:
Following up, it is correct that it should be an error (replacing the existing exception). Will fix it.

---
In reply to: [105035896](https://github.com/dotnet/roslyn/pull/17113#discussion_r105035896) [](ancestors = 105035896,101839466)